src/java/edu/slu/tpen/entity/image/Canvas.java

```diff
@@ -254,63 +255,63 @@ public static JSONArray getLinesForProject(Integer projectID, String canvasID, I
+//            Project proj = new Project(projectID);
+//            String projName = proj.getName();
+//            User u = new User(UID);
+//            Folio[] folios = proj.getFolios();
 //            System.out.println(folios.length);
-            for (Folio f : folios) {
+//            for (Folio f : folios) {
 //                pageList.add(JsonHelper.buildPage(projectID, projName, f, u,"v3"));
 //          ÷        pageList.add(JsonHelper.buildPage(f)); 
-                  pageList.add(JsonHelper.buildPage(f, "v3"));
-            }
-            System.out.println("Put all canvas together");
-            annotationPage.put("resources", pageList);
-            annotations = getAnnotationSet(projectID,folioNumber);
+//                  pageList.add(JsonHelper.buildPage(f, "v3"));
+//            }
+//            System.out.println("Put all canvas together");
+//            annotationPage.put("resources", pageList);
+//            annotations = getAnnotationSet(projectID,folioNumber);
 //            System.out.println(Arrays.toString(annotations));
+
         
-        }
-        annotationPage.put("items",annotations);
-        //annotationList.element("@context", "http://iiif.io/api/presentation/2/context.json");
-        //annotationList.element("testing", "msid_creation");
-        
+//        }
+        annotationPage.put("items",annotations); 
         Transcription[] lines;
         lines = getProjectTranscriptions(projectID, folioNumber); //Can return an empty array now.
         int numberOfLines = lines.length;
         List<Object> resources = new ArrayList<>();
-        //System.out.println("How many lines?   "+numberOfLines);
         for (int i = 0; i < numberOfLines; i++) { //numberOfLines can be 0 now.
             if (lines[i] != null) {
-                //System.out.println("On line "+i);
                 dateString = "";
-                //when it breaks, it doesn't get this far
-                //System.out.println(lines[i].getLineID() + " " +lines[i].getDate().toString());
                 int lineID = lines[i].getLineID();
                 Map<String, Object> lineAnnot = new LinkedHashMap<>();
                 String lineURI = "line/" + lineID;
                 String annoLineID = getRbTok("SERVERURL") + "line/" + lineID;
-                //lineAnnot.put("@id", lineURI);
                 lineAnnot.put("id", annoLineID);
-                lineAnnot.put("_tpen_line_id", lineURI);
-                lineAnnot.put("type", "AnnotationPage");
-                lineAnnot.put("motivation", "oad:transcribing");
-                lineAnnot.put("resource", buildQuickMap("@type", "cnt:ContentAsText", "cnt:chars", encoder().decodeForHTML(lines[i].getText())));
-                lineAnnot.put("on", format("%s#xywh=%d,%d,%d,%d", canvasID, lines[i].getX(), lines[i].getY(), lines[i].getWidth(), lines[i].getHeight()));
+                lineAnnot.put("type", "Annotation");
+                lineAnnot.put("motivation", "transcribing");
```

Comment:
@thehabes Did we decide on what this should be? I don't think we have it in our context and IIIF only gets close with "supplementing".